echopype/convert/api.py

```diff
@@ -347,19 +343,11 @@ def open_raw(
         and need to be added to the converted file
     storage_options : dict, optional
         options for cloud storage
-    use_swap: bool
-        **DEPRECATED: This flag is ignored**
-        If True, variables with a large memory footprint will be
-        written to a temporary zarr store at ``~/.echopype/temp_output/parsed2zarr_temp_files``
-    destination_path: str
-        The path to a swap directory in a case of a large memory footprint.
-        This path can be a remote path like s3://bucket/swap_dir.
+    use_swap: "auto" | bool
+        Flag to use disk swap in a case of a large memory footprint.
```

Comment:
Somehow I could not make suggestions on the lines below since it includes deleted lines, so I put the whole thing here:
```suggestion
    use_swap: "auto" | bool
        Flag to use disk swap in case of a large memory footprint. 
        When set to True (or when set to "auto" and large memory footprint is needed, 
        this function will create a temporary zarr store at the operating system's 
        temporary directory.
```